packages/@vuepress/theme-default/src/client/styles/vars.scss

```diff
@@ -65,6 +65,17 @@
   --sidebar-width-mobile: calc(var(--sidebar-width) * 0.82);
   --content-width: 740px;
   --homepage-width: 960px;
+
+  // transition vars
+  --t-options: 0.3s ease;
+  --t-all: all var(--t-options);
+  --t-opacity: opacity var(--t-options);
+  --t-color: color var(--t-options);
+  --t-border-color: border-color var(--t-options);
+  --t-background-color: background-color var(--t-options);
+  --t-color-all: var(--t-color), var(--t-border-color),
+    var(--t-background-color);
+  --t-transform: transform var(--t-options);
```

Comment:
One more thing: there are two different naming patterns

- `--t-${propertyName}`: `--t-all`, `--t-opacity`, `--t-color`
- `--t-${somethingElse}`: `--t-options`, `--t-color-all`

Any good idea to normalize the naming pattern?

Reply:
Maybe an extra prefix `--t-p` for property name? 🤔

Reply:
10fe60f